### ISSUE TITLE
Do I need to normalize the input data?

### ISSUE THREAD

**Issue by fmenat** [gastruc/AnySat#3]
Hi, thanks for the repo!

I want to know if I need to normalize the sensor data before feeding into AnySat, or what range of values is expecting the model: [0-1], [0-255] for RGB, standard normalization?  I could not find this information.

Thanks!

**Comment by gastruc** [comment]
Hi,
Indeed, you need to normalize the data in a standard way. If you wish, I could provide some generic normalization values

**Comment by fmenat** [comment]
Ah okay, thanks!

I was doing standard normalization (subtract mean and divide by standard deviation) but I wasn't sure if it was the right way to use it with this model.

Thanks again.

**Comment by DeepT7** [comment]
@gastruc Hi,
 Thank you for this great project. I am trying to fine-tune the model on the Sickle dataset using the command "python src/train.py exp=Sickle_AnySat_FT". I did not do any preprocessing steps on the raw Sickle data because I believe it was already in the "src/data/Sickle.py"  file. I stopped at the 50th epoch, achieving mIoU = 0.74, however when I visualized the prediction result and compared it with the ground truth, they differed a lot. 
Do I need to do data preprocessing before training, or can you provide the code to visualize the model's prediction results on the Sickle dataset? Thank you very much.

**Comment by DeepT7** [comment]
> Hi, Sickle is particularly challenging to visualize due to the proportion of -1 values in the mask, which is excluded from our metrics and loss. These regions make the results difficult to interpret, as predictions can vary within them. To address this, you can replace all -1 mask pixels with a specific value, such as 0, during visualization.

Thank you, I really appreciate your advice.  I noticed the -1 values ​​in ground truths so I added them with 1 to solve the problem but still didn't get good results. Another problem, when running inference the result is an array of size (2, 32, 32). I used the function numpy.argmax(axis = 0) to convert it into a binary mask. Is this method correct? I would be very grateful if you would help me clarify these issues.

**Comment by tcourat** [comment]
> Hi, Indeed, you need to normalize the data in a standard way. If you wish, I could provide some generic normalization values

What would be the normalization values for SPOT images ? Also, what tile size should be used for inference ?
